package.json

```diff
@@ -1,17 +1,18 @@
 {
   "name": "axios-case-converter",
-  "version": "0.3.1",
+  "version": "0.4.0",
   "main": "lib/index.js",
   "module": "es/index.js",
   "scripts": {
-    "lint-src": "eslint src",
-    "lint-examples": "eslint examples",
-    "test": "npm run lint-src && cross-env BABEL_ENV=cjs babel-node test/index.js | tap-spec",
-    "test-without-lint": "cross-env BABEL_ENV=cjs babel-node test/index.js | tap-spec",
-    "build:cjs": "cross-env BABEL_ENV=cjs babel src --out-dir lib",
-    "build:es": "cross-env BABEL_ENV=es babel src --out-dir es",
+    "check": "tsc --noEmit",
+    "lint": "eslint {src,test}/**/*.ts",
+    "lint:fix": "eslint {src,test}/**/*.ts --fix",
+    "build:cjs": "BABEL_ENV=cjs babel src --out-dir lib --extensions .ts",
+    "build:es": "BABEL_ENV=es babel src --out-dir es --extensions .ts",
+    "build:typings": "tsc --outDir typings/ --emitDeclarationOnly",
```

Comment:
TypeScript can not found `typings/index.d.ts`. It is needed to add `"types": "typings/index.d.ts"` or set `--outDir` to `lib` and `es`.

Reply:
ref. https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/declaration%20files/Publishing.md#including-declarations-in-your-npm-package